pkg/middleware/logging/logging.go

```diff
@@ -144,6 +146,19 @@ func reportable(l logger.Logger) interceptors.CommonReportableFunc {
 			fields = append(fields, zap.String(userAgentKey, userAgent))
 		}
 
+		if c.Service == grpcHealth {
```

Comment:
I tested this PR via adding 

```
- OPENFGA_SILENT_HEALTHCHECK=true
```

to https://github.com/openfga/openfga/blob/main/docker-compose.yaml#L43

and then running `docker-compose up --build`.

In the logs I see

` starting openfga service... "SilentHealthChecks":true`

but then this `if` doesn't seem to be doing anything because `silentHealthChecks is `false`.